### PR TITLE
Categories entity

### PR DESCRIPTION
**Pull request by szewa-91**
Hi. First entity added, categories are now available by REST service. Run the application and then go to `localhost:8080/categories`. You can run the application with db-init (details in README) to have DB initially populated.

### CODE REVIEW

**Review by ppkt — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 
 
-## Requirements
+## Database
 To run the project, you need to install and run an SQL server. The most common (and advised) solution is MySQL 
 ([get from website](https://www.mysql.com/downloads/)) or download using your package manager.
 
 Then, you have to create a table (named for example `srpr`). Also create a user and grant them all privileges on this table
```
Comment (qwrtln) [inline README.md]: Are you sure it has to be a table? As I recall an empty DB without any tables was fine.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,20 @@ mvn install
 mvn spring-boot:run
 ```
 
+### Init db on run
+On default, the application will only create the schema of database if not yet created.
```
Comment (qwrtln) [inline README.md:36]: if **it is** not yet created
If it's about the creation of the schema, it should be like this.

File: src/main/resources/data.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+INSERT INTO category (name, icon) values ( 'Las', 'las.jpg');
+INSERT INTO category (name, icon) values ( 'Owady', 'zuk.jpg');
+INSERT INTO category (name, icon) values ( 'Miasto', 'miasto.jpg');
+INSERT INTO category (name, icon) values ( 'Ognicho', 'bonfire.jpg');
+INSERT INTO category (name, icon) values ( 'Morze', 'wave.jpg');
+INSERT INTO category (name, icon) values ( 'Marsjanie', 'outer.jpg');
```
Comment (qwrtln) [inline src/main/resources/data.sql:7]: :joy:

File: README.md (hunk 2 above)
Comment (qwrtln) [reply README.md:36]: And it's "By default"

**Review by qwrtln — CHANGES_REQUESTED** [review]

**Pull request merged**